docs/output.md

```diff
@@ -1,35 +1,105 @@
-# nf-core/mag
-Assembly, binning and annotation of metagenomes
+# nf-core/mag: Output
 
 This document describes the output produced by the pipeline. Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline.
 
 ## Pipeline overview
+
 The pipeline is built using [Nextflow](https://www.nextflow.io/)
 and processes data using the following steps:
 
-* [FastQC](#fastqc) - read quality control
-* [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
+- [FastQC](#fastqc) - read quality control
+- [fastp](#fastp) - read trimming
+- [megahit](#megahit) - assembly
+- [quast](#quast) - assembly quality report
+- [MultiQC](#multiqc) - aggregate report, describing results of the whole pipeline
+- [metabat](#metabat) - recovered draft genome bins from assembled metagenomes
+- [checkm](#checkm) - genome bins quality control and merging of compatible bins
+- [refinem](#refinem) - improving metagenome-assembled genome bins
 
 ## FastQC
+
 [FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your reads. It provides information about the quality score distribution across your reads, the per base sequence content (%T/A/G/C). You get information about adapter contamination and other overrepresented sequences.
 
 For further reading and documentation see the [FastQC help](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
 
+> **NB:** The FastQC plots displayed in the MultiQC report shows both _untrimmed_ and _trimmed_ reads.
```

Comment:
Is this useful? In other pipelines I usually hide the trimmed FastQC files from the MultiQC report. It's useful when digging in to find problems, but generally I find that it doesn't really add anything for normal projects.